### ISSUE TITLE
isFirstOlder uses string-type comparison instead of integer-type

### ISSUE THREAD

**Issue by pmikolajek** [logux/logux#43]
Simple reproduction code:

```javascript
import { isFirstOlder } from '@logux/core/is-first-older';

const firstMeta = {
    id: "1592781200022 server:hwP9fpM0 9",
    time: 1592781200022,
};

const secondMeta = {
    id: "1592781200022 server:hwP9fpM0 10",
    time: 1592781200022,
};

console.log(isFirstOlder(firstMeta, secondMeta));  // prints true instead of false
```

When two messages have the same timestamp, the IDs are compared. If one of those has more _digits_ than the other, then the comparison breaks—because `9 < 10`, but `"9" > "10"`.

This leads to incorrect behaviour in the case of more than 10 messages being sent in the same millisecond.

**Comment by ai** [comment]
Fixed d5a82309223e1a126d10a91a433b0313e1f6d692

**Comment by ai** [comment]
Released in 0.6.1
